app/views/course/assessment/answer/_comments.html.slim

```diff
@@ -1,5 +1,5 @@
 = render partial: answer.discussion_topic,
          locals: { \
-           post_partial: 'course/assessment/answer/comment', \
+           post_partial: 'course/discussion/post', \
```

Comment:
There is a `display_topic` helper, you can use that here. 

I don't remember why I didn't update it here.